docs/source/developers/contributing.rst

```diff
@@ -168,11 +274,15 @@ remote repo still holds the old history, you would need to do a force push. ::
 look at your updates, please ensure you comment on the PR on GitHub as simply force
 pushing does not trigger a notification in the GitHub user interface.
 
-Simplifying ``rebase``
-++++++++++++++++++++++
+Also, once you have a pull request up, be sure you pull from ``origin``
+before rebasing and force-pushing. Arrow maintainers can push commits directly
+to your branch, which they sometimes do to help move a pull request along.
+In addition, the GitHub PR "suggestion" feature can also add commits to
+your branch, so it is possible that your local copy of your branch is missing
+some additions.
```

Comment:
I'm not a fan of the "suggestion" feature and don't want to encourage its use

Reply:
I don't think this paragraph is necessarily encouraging it, just mentioning how to resolve it if some reviewers use it

(I personally like the feature, not necessarily to then commit through the github interface, but just as a means to give a much more concrete review comment, instead of trying to describe in words what you would propose to change)

Reply:
FWIW I too like the suggestion feature: I find it makes for a better being-reviewed experience because trivial or pedantic suggestions can just be accepted. In any case, I agree with Joris that I don't read this paragraph as an endorsement, just an acknowledgement of a way that `origin` may have commits that you don't have locally, even though it is "your" branch.